src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,65 @@
+package duke;
+
+import duke.command.Command;
+
+/**
+ * The Duke program implements a chatbot that
+ * helps the user to keep track of various things.
+ * <p>
+ * The program can take in tasks and add it to a
+ * task list, list it out, delete tasks,
+ * mark/unmark tasks and save the tasks.
+ *
+ * @author fungusta
+ * @version 0.1
+ * @since 2022-08-23
+ */
+public class Duke {
+
+    private final Storage storage;
+    private final TaskList taskList;
+    private final Ui ui;
+    private boolean isBye; //Indicates if bye was input by user
```

Comment:
```suggestion
    private boolean isBye; // Indicates if bye was input by user
```

Should there be a space between the `//` and the comment? I think the space is required in many coding standards, although not specifically CS2103T's.

This also applies to more inline comments below.